table/block_fetcher.h

```diff
@@ -76,9 +76,16 @@ class BlockFetcher {
   inline CompressionType get_compression_type() const {
     return compression_type_;
   }
+  inline CompressionType get_raw_compression_type() const {
```

Comment:
"raw" has a history of being a misleading term, and I believe the distinct compression types are not needed, base on the single usage of get_compression_type(). Seems very possible to simplify.